docs/user-guide/gui/projects.md

```diff
@@ -0,0 +1,22 @@
+# Projects
+Projects are OFRAK's way of saving, organizing, and sharing your work. Projects are collections of binaries and scripts along with metadata that describes relationships between them.
+
+## The Interface
+The Project Options interface is available from the start menu and gives you three options for opening a project. The "Create New Project" opens up a blank project, "Open Existing Project" will open a project already in your Project Path, and "Clone Project from Git" clones the URL provided to your Project Path. The Project Path, where all of the projects are saved to and loaded from, may be modified in the Advanced Options section of the Project Options interface.
+
+![](assets/project-options.png)
+
+The OFRAK Project Manager interface is loaded upon opening a project and displays the structure of the project along with options to modify it. The Binaries panel displays each binary in the project. Select a binary to work with by clicking on its name. The Script panel shows options for each script in your project. Upon selecting a binary, two checkboxes options appear for each script. Hovering over each checkbox describes its function. One check box selects whether the script is compatible with the currently selected binary, allowing access to it in the Run Script option of the OFRAK Resource toolbar. The second checkbox selects a script to be run upon entering the OFRAK Resource interface, giving a starting point for exploration.
```

Comment:
```suggestion
The OFRAK Project Manager interface is loaded upon opening a project. It displays the structure of the project along with options to modify it. The Binaries panel displays each binary in the project. Select a binary to work with by clicking on its name. The Script panel shows options for each script in your project. Upon selecting a binary, two checkboxes options appear for each script. Hovering over each checkbox describes its function. One check box selects whether the script will run with the currently selected binary, allowing access to it in the Run Script option of the OFRAK Resource toolbar. The second checkbox selects a script to be run upon entering the OFRAK Resource interface, giving a starting point for exploration.
```

Reply:
changed to "can be run on" because i thin "will" makes it sound like it will run on startup